docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -405,11 +408,26 @@ endif::[]
 EOF
 ----
 +
-* Prepare adopted EDPM workloads to use Ceph backend for Block Storage service (cinder), if configured so
+<1> If TLS Everywhere is enabled, change `spec:tlsEnabled` to `true`.
+<2> If using a custom DNS Domain, modify the `spec:nodes:[NODE NAME]:hostName` to use fqdn for the node.
+<3> Replace `<bridge_mappings>` with the value of the bridge mappings in your configuration, for example, `"datacentre:br-ctlplane"`.
+
+. Ensure that the `ovn-controller` settings that are configured in the `OpenStackDataPlaneNodeSet` CR are the same as were set in the Compute nodes before adoption. This configuration is stored in the `external_ids` column in the `Open_vSwitch` table in the Open vSwitch database:
++
+----
+ovs-vsctl list Open .
+...
+external_ids        : {hostname=standalone.localdomain, ovn-bridge=br-int, ovn-bridge-mappings=<bridge_mappings>, ovn-chassis-mac-mappings="datacentre:1e:0a:bb:e6:7c:ad", ovn-encap-ip="172.19.0.100", ovn-encap-tos="0", ovn-encap-type=geneve, ovn-match-northd-version=False, ovn-monitor-all=True, ovn-ofctrl-wait-before-clear="8000", ovn-openflow-probe-interval="60", ovn-remote="tcp:ovsdbserver-sb.openstack.svc:6642", ovn-remote-probe-interval="60000", rundir="/var/run/openvswitch", system-id="2eec68e6-aa21-4c95-a868-31aeafc11736"}
+...
+----
++
+* Replace `<bridge_mappings>` with the value of the bridge mappings in your configuration, for example, `"datacentre:br-ctlplane"`.
+
+. If you use a Ceph back end for {block_storage_first_ref}, prepare the adopted data plane workloads:
```

Comment:
Maybe:
"If you use a Ceph back end for {block_storage_first_ref}, you must prepare the adopted data plane workloads:"

(if this is true)